### PR TITLE
SPCTR-3346 Fix: Global - Removed the Block Editor Spacing setting for FSE themes.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3346

### Screenshots

- https://d.pr/v/yL3lM0

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added in card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,13 +215,16 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
+* Fix: Global - Removed the Block Editor Spacing setting for FSE themes.
+
 ### 2.7.9 - WEDNESDAY, 13th SEPTEMBER 2023 ###
 * Improvement: Info Box - Changed the Info Box icon position dropdown to a multi-control.
 * Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
 * Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended.
 * Fix: Image Gallery - Resolved an issue wherein the lightbox would not work as intended within Containers.
 * Fix: Post Blocks - Resolved an issue wherein text-formatting for buttons would appear as HTML tags on the front-end.
-* Fix: Price List - Resolved an issue in which the title bottom margin, row, and column gaps would not work as intended in the responsive editor.
+* Fix: Price List - Resolved issues wherein the title bottom margin, and the row and column gaps would not work as intended for responsive devices.
```
Comment (bsf-zanev) [inline README.md:227]: We don't need this change here.

Comment (AbhijitNage123) [reply README.md]: Resolved

File: admin-core/assets/src/dashboard-app/pages/settings/Settings.js [hunk 2]
```diff
@@ -122,7 +122,10 @@ const Settings = () => {
 										<DynamicContent />
 									)
 								}
-                                <BlocksEditorSpacing/>
+								{ ! uag_react.is_block_theme && (
+									<BlocksEditorSpacing />
+									)
+								}
```
Comment (bsf-zanev) [inline admin-core/assets/src/dashboard-app/pages/settings/Settings.js:128]: Quick Improvement.
```suggestion
								{ ! uag_react.is_block_theme && (
									<BlocksEditorSpacing />
									) }
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just one minor coding improvement.

File: classes/class-uagb-init-blocks.php [hunk 3]
```diff
@@ -776,6 +776,7 @@ function( $a, $b ) {
 				'current_post_id'                         => get_the_ID(),
 				'btn_inherit_from_theme'                  => UAGB_Admin_Helper::get_admin_settings_option( 'uag_btn_inherit_from_theme', 'disabled' ),
 				'wp_version'                              => get_bloginfo( 'version' ),
+				'is_block_theme'                          => ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ),
```
Comment (adi3890) [inline classes/class-uagb-init-blocks.php:779]:
```suggestion
				'is_block_theme'                          => UAGB_Admin_Helper::is_block_theme,
```

i see we are usin this reperative code at multiple places can be replace it with a similar helper function.

File: admin-core/assets/src/dashboard-app/pages/settings/Settings.js (hunk 2 above)
Comment (AbhijitNage123) [reply admin-core/assets/src/dashboard-app/pages/settings/Settings.js]: Resolved

File: classes/class-uagb-init-blocks.php (hunk 3 above)
Comment (AbhijitNage123) [reply classes/class-uagb-init-blocks.php]: Done. Please review

**Review by adi3890 — APPROVED** [review]

**Pull request merged**